core/storage/base_model/gae_models.py

```diff
@@ -380,11 +379,7 @@ class BaseCommitLogEntryModel(BaseModel):
     """Base Model for the models that store the log of commits to a
     construct.
     """
-    # Update superclass model to make these properties indexed.
-    created_on = ndb.DateTimeProperty(auto_now_add=True, indexed=True)
-    last_updated = ndb.DateTimeProperty(auto_now=True, indexed=True)
 
-    # The id of the user.
```

Comment:
Every attribute needs a one-line comment above it stating what it is. Don't delete it.

Reply:
Just a mistake, fixed.